src/services/connection/connection.transport.ts

```diff
@@ -0,0 +1,6 @@
+export class ConnectionTransport {
```

Comment:
Could you design your ConnectionTransport as a 'value' object. You can use ConnectionStatus/ConnectionStatuses as a example. I use this design because signalr connection status is basically a string that has only limited range of values. This design prevents the api user from filling in something random